src/App.jsx

```diff
@@ -14,6 +15,59 @@ export const goods = [
   'Garlic',
 ];
 
+export const Good = ({ good }) => {
+  const [buttonClass, setButtonClass] = useState('button');
```

Comment:
all this states are redundant
create one instead
const [selectedGood, setSelectedGood] =